bindings/nodejs/lib/types/models/api/block-response.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { BlockState } from '../state';
+import { BlockFailureReason } from './block-failure-reason';
+import { Block, BlockId } from '../../block';
+import { TransactionMetadataResponse } from './transaction-response';
+
+/**
+ * Response from the metadata endpoint.
+ */
+export interface BlockMetadataResponse {
+    /**
+     * The block id.
+     */
+    blockId: BlockId;
+    /**
+     * The block state.
+     */
+    blockState: BlockState;
+    /**
+     * The block failure reason.
+     */
+    blockFailureReason?: BlockFailureReason;
+    /**
+     * The metadata of the transaction in the block.
+     */
+    transactionMetadata?: TransactionMetadataResponse;
+}
+
+/**
+ * Response from the full endpoint.
+ */
+export interface BlockFullResponse {
```

Comment:
What is this? Doesn't seem to exist in python/rust. Is that supposed to be `BlockWithMetadataResponse`?

Reply:
good catch ... it's from the TIP ... but I think I forgot to update the others. I just saw I even had it in my todo list.

Reply:
FWIW, BlockFullResponse is a terrible name

Reply:
I got confirmation that this will be renamed, so I'll change it to `BlockWithMetadataResponse`.